packages/amplify-provider-awscloudformation/src/push-resources.js

```diff
@@ -168,6 +192,110 @@ async function run(context, resourceDefinition) {
   }
 }
 
+async function createEnvLevelConstructs(context) {
+  const { StackName: stackName } = context.amplify.getProjectMeta().providers[constants.ProviderName];
+
+  const hasContainers = envHasContainers(context);
+
+  const updatedMeta = {};
+
+  Object.assign(updatedMeta, await createNetworkResources(context, stackName, hasContainers));
+
+  context.amplify.updateProvideramplifyMeta(providerName, updatedMeta);
+}
+
+async function createNetworkResources(context, stackName, needsVpc) {
```

Comment:
I think it would better if we either have a separate util in this package for this function or an exported function from the api category and not merge this into this specific file.